src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -673,5 +674,47 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         }    
         
     }
-    
+
+    /**
+     * @description Detect if any opportunities exists where the same contact
+     * is associated with that opportunity multiple times through multiple
+     * opportunity contact role records.
+     */
+     public void verifyNoDuplicateContactRoles() {
+        try {
+            List<AggregateResult> results = [
+                SELECT
+                    COUNT(Id),
+                    OpportunityId,
+                    ContactId
+                FROM OpportunityContactRole
+                GROUP BY OpportunityId, ContactId
+                HAVING Count(Id) > 1
```

Comment:
@dospromptman @mpusto I thought we would only complain about duplicate OCR's whose role was in a soft credit role.  If we don't, we might find that we are giving this error to lots of orgs who are using some custom non-credit OCR for some sort of tracking.  I think this soql just needs to add a where clause of Role in :softCreditRoles.